app/params/encoding.go

```diff
@@ -0,0 +1,15 @@
+package params
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+)
+
+// EncodingConfig specifies the concrete encoding types to used for regen app.
+type EncodingConfig struct {
```

Comment:
while removing the `MakeTestEncodingConfig` is desired, I believe copy/pasting this struct is not needed: we can use simapp's one directly.